flow-cache-updates-lambda/lib/cacheOps.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @module cacheOps
+ * exports a single function used to get a redis command for an object
+ */
+
+const keyGen = require('./keyGen');
+const constants = require('./constants');
+
+const cacheOps = {};
```

Comment:
Correct.. You're not using this it just makes it easier on the eyes :) If that were important then you'd probably wanna use `class` and make it explicit what kind of object it is.